modules/AM_TaskTemplates/language/en_us.lang.php

```diff
@@ -80,4 +80,6 @@
     'LBL_UTILIZATION' => 'Utilization (%)',
     'LBL_DURATION' => 'Duration (days)',
     'LBL_AM_TASKTEMPLATES_AM_PROJECTTEMPLATES_FROM_AM_PROJECTTEMPLATES_TITLE' => 'Project Templates',
+
+    'LBL_AM_TASKTEMPLATES_AM_PROJECTTEMPLATES_FROM_AM_TASKTEMPLATES_TITLE' => 'Task Templates: Project Templates from Task Templates Title',
```

Comment:
No, is exact long as suppose to be, also if @horus68 found just one sting he is not happy, that means to me i did good job. Expect star in my profile :) or at least medal :)